Integrations/integration-Pwned.yml

```diff
@@ -182,7 +189,7 @@ script:
     - contextPath: Account.Email.Compromised.Vendor
       description: For comporomised emails, the vendor that made the decision
       type: string
-    - contextPath: Account.Email.Reason
+    - contextPath: Account.Email.Compromised.Reason
```

Comment:
since we are changing the path anyway, maybe we can add a more meaningful name than reason?
or is it the agreed convention for us to say "who found this?"
@kirbles19 lets check this.

Reply:
@Arsenikr proposed to use "reporters"

Reply:
Still relevant?

Reply:
@kirbles19  no ty, we're good